### ISSUE TITLE
Add Cloudinary folder route

### ISSUE THREAD

**Issue by Corentin7301** [kevcomparadise/directus-vue3-cloudinary-upload#2]
Hi ! 
I would like to use your module but I need to choose an upload folder in Cloudinary.
Do you think it is possible to add this functionality?
Thanks!

**Comment by kschnekenburger** [comment]
Hi @Corentin7301 I'll see what I can do

**Comment by kschnekenburger** [comment]
Why you delete file extension ?
Thanks !

**Comment by kschnekenburger** [comment]
#5 Does this meet your need? @Corentin7301

**Comment by Corentin7301** [comment]
Hi @kevcomparadise !
Thanks for your work!
The new file management is great!
But I've always had the same problem for which I had removed the file extension.
In database, the image name has double extensions like: `-------/blobs.png.png` .
I don't know why I've this bug. It's for that I've deleted the file extension.
In Cloudinary, the file has a good simple extension like `-------/blobs.png`.
Thanks for reply!
